quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/jaxrs/QuarkusKeycloakApplication.java

```diff
@@ -106,9 +104,4 @@ public void createTemporaryMasterRealmAdminService(String clientId, String clien
         new ApplianceBootstrap(session).createTemporaryMasterRealmAdminService(clientId, clientSecret /*, adminExpiration*/);
     }
 
-    private String getEnvOrProp(String envKey, String propKey) {
```

Comment:
We no longer want to support the property based value? Or is this handled implicitly by `getOptionalKcValue()`?

Reply:
Adding the system property was just a temporary thing for the PoC testsuite as we couldn't use the environment variable with the embedded mode. With the new bootstrapping we now have a proper way of using cli-args (also supports env variables or sys properties) instead, so no longer need this workaround.

It was included in 25, but not documented, so pretty sure no-one knew about it ;)

Reply:
Thanks for clearing that up